### DISCUSSION TITLE
Making a missing target: warning to the user?

### DISCUSSION THREAD

**Discussion by llrs** [ropensci/targets#923] [General]
Below a reprex for a new project:
```r
library("targets")
packageVersion("targets")
## [1] '0.13.1'
targets::use_targets()
## • Writing file "_targets.R".
## • Writing file "run.R".
## • Writing file "run.sh".
## • Modify '_targets.R'
tar_make("testing_target")
## • end pipeline: 0.225 seconds
```

It would be nice if there were some kind of message displayed when a missing target is requested.
I tried with `reporter = "verbose_positives"` but it didn't add anything new.
This rose when I changed some targets (or copied over a very similar project) and didn't do a search and replace everywhere and when asking for a specific target I missed that. I stumbled a bit on why `tar_make` run successfully but then my other target didn't finish correctly.

**Comment by wlandau** [comment]
Sure. Effective 4f44c7729dcbd215977e8c3ebc9c077ac4e23295, `targets` prints a "no targets found" message when there are no targets in the pipeline after accounting for the `names` argument. (Unfortunately it isn't feasible to detect specific requested targets that aren't found in the general case because `tidyselect` is the main driver of the `names` argument.)

**Reply by llrs** [reply]
Thanks!!
